test/stat_tracker_test.rb

```diff
@@ -106,15 +106,27 @@ def test_it_has_a_rival
   # ************* Season Statistics *************
 
   def test_winningest_coach
-    assert_equal "Claude Julien", @stat_tracker.winningest_coach(game_ids)
-    assert_equal "Alain Vigneault", @stat_tracker.winningest_coach(game_ids)
+    assert_equal "Claude Julien", @stat_tracker.winningest_coach("20132014")
+    assert_equal "Alain Vigneault", @stat_tracker.winningest_coach("20142015")
```

Comment:
Thanks for updating these arguments.

Reply:
Np @JesseMellinger - let me know if/when you're ready to merge